apple-pie/index.html

```diff
@@ -4,18 +4,62 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+    <header>      
+      <h1>Apple Pies</h1>
+      <img scr="/images/apple-pie.jpg"/>
+            
+    <p class="container">This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
     be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
+    favorites as well!</p>
+
+  </header>
+    
+    <img scr="/images/recipe-info.png"/>
+
+    
+  
+    </section>
+
+    <section id="Ingridients" class="container">
+       
+      <h2>Ingredients</h2>       
+        
+      <ul>    
+    
+    <li>1 recipe pastry for a 9 inch double crust pie</li>
+    <li>1/2 cup unsalted butter</li> 
+    <li>3 tablespoonsall-purpose flour</li>  
+    <li>1/4 cup water</li> 
+    <li>1/2 cup white sugar</li> 
+    <li>1/2 cup packed brown sugar</li> 
+    <li>8 Granny Smith apples - peeled, cored and sliced</li> 
+
+   </ul>
```

Comment:
Try to maintain the indentation of your code, it'll help to read the code easier in the future